src/axom/slam/tests/slam_map_BivariateMap.cpp

```diff
@@ -390,7 +390,7 @@ void constructAndTestBivariateMapIterator(int stride)
   SLIC_INFO("Checking the elements with BivariateMap flat iterator.");
   {
     auto iter = m.begin();
-    auto flat_idx = 0;
+    // auto flat_idx = 0;

```

Comment:
Consider removing the unused variable.

Reply:
I wasn't sure if they were there for a reason.

@publixsubfan -- should we keep these variables in the `slam_map_BivariateMap` tests? It seems like you used them at some point.

Reply:
I'll leave these for now and we can remove them in the future